internal/hardware/validator.go

```diff
@@ -173,6 +175,27 @@ func (v *validator) GetClusterHostRequirements(ctx context.Context, cluster *com
 	}, nil
 }
 
+func (v *validator) GetInfraEnvHostRequirements(ctx context.Context, infraEnv *common.InfraEnv, host *models.Host) (*models.ClusterHostRequirements, error) {
+	/*
```

Comment:
I don't think that a new code block should have code in comments

Reply:
fixed